tests/dist-check.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
Also, it can't go in cherrypy/test as that causes cherrypy to be imported too early.